aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -58,6 +58,25 @@ class CreateWalletRequestSchema(OpenAPISchema):
         ),
     )
 
+    wallet_dispatch_type = fields.Str(
+        description="Webhook target dispatch type for this wallet. \
+            default - Dispatch only to webhooks associated with this wallet. \
+            base - Dispatch only to webhooks associated with the base wallet. \
+            both - Dispatch to both webhook targets.",
+        example="default",
+        default="default",
+        validate=validate.OneOf(["default", "both", "base"]),
+    )
+
+    wallet_webhoook_urls = fields.List(
```

Comment:
Typo here. wallet_webhoook_urls -> wallet_webhook_urls.

Reply:
@baegjae Thanks for the feedback, I have fixed the typos and adjusted the redundant logic.